{
  "page": {
    "id": "mxfydrt8b8xw",
    "name": "LinkedIn API",
    "url": "https://www.linkedin-apistatus.com",
    "time_zone": "Etc/UTC",
    "updated_at": "2025-01-13T17:20:31.400Z"
  },
  "components": [],
  "incidents": [
    {
      "id": "fk1wj964rzts",
      "name": "LinkedIn AccessToken and Authorization API Returning 429 Error",
      "status": "investigating",
      "created_at": "2025-01-13T16:22:37.661Z",
      "updated_at": "2025-01-13T16:22:37.691Z",
      "monitoring_at": null,
      "resolved_at": null,
      "impact": "none",
      "shortlink": "https://stspg.io/shhfb88jmcmy",
      "started_at": "2025-01-13T16:22:37.655Z",
      "page_id": "mxfydrt8b8xw",
      "incident_updates": [
        {
          "id": "1d55k1nj7bpt",
          "status": "investigating",
          "body": "We are currently experiencing an issue across the platform that may cause some API requests to /v2/accessToken and /v2/authorization to return a 429 error code. We are investigating the issue and working on a resolution.",
          "incident_id": "fk1wj964rzts",
          "created_at": "2025-01-13T16:22:37.689Z",
          "updated_at": "2025-01-13T16:22:37.689Z",
          "display_at": "2025-01-13T16:22:37.689Z",
          "affected_components": null,
          "deliver_notifications": false,
          "custom_tweet": null,
          "tweet_id": null
        }
      ],
      "components": [],
      "reminder_intervals": null
    }
  ],
  "scheduled_maintenances": [],
  "status": {
    "indicator": "none",
    "description": "All Systems Operational"
  }
}
